src/bundle/Resources/public/js/scripts/core/custom.dropdown.js

```diff
@@ -133,14 +133,19 @@
 
         getItemsContainerHeight(isItemsContainerTop) {
             const DROPDOWN_MARGIN = 16;
+            const MODAL_SELECTOR = '.modal[aria-modal=true]';
+            const documentElementHeight = document.documentElement.getBoundingClientRect().height;
+            const itemsContainerHeight = this.itemsContainer.getBoundingClientRect().top;
```

Comment:
```suggestion
            const itemsContainerTop = this.itemsContainer.getBoundingClientRect().top;
```

Reply:
In general, I agree with this naming change. But in this very case: wouldn't this be a little misleading given, we pass `isItemsContainerTop` parameter already? As for me, it would be harder to figure out what the code does at first glance.

Reply:
Naming `getBoundingClientRect().top` `height` is even worse in understanding what this variable stores. Top position is completely different than height.
Maybe `itemsContainerTopPosition`?
Or renaming `isItemsContainerTop` as seems like this is not very descriptive either?

Reply:
WDYT @dew326?
https://github.com/ezsystems/ezplatform-admin-ui/pull/1709/commits/ef4eaa409ef77ed53a68cc69e9c4f1d26f2b5887